recipes/oatpp-postgresql/all/test_package/CMakeLists.txt

```
@@ -1,11 +1,11 @@
 cmake_minimum_required(VERSION 3.1)
-project(oatpp-postgresql-test CXX)
+project(test_package CXX)
 
 include(${CMAKE_BINARY_DIR}/conanbuildinfo.cmake)
 conan_basic_setup(TARGETS)
 
 find_package(oatpp-postgresql REQUIRED)
 
-add_executable(oatpp-postgresql-test oatpp-postgresql-test.cpp)
+add_executable(${PROJECT_NAME} test_package.cpp)
 set_property(TARGET ${PROJECT_NAME} PROPERTY CXX_STANDARD 11)
-target_link_libraries(oatpp-postgresql-test PRIVATE oatpp::oatpp-postgresql)
+target_link_libraries(${PROJECT_NAME} PRIVATE CONAN_PKG::oatpp-postgresql)
```

Since this project has cmake support and exports _simple_ namespace targets, we like to have the official value in the test_package

https://github.com/oatpp/example-postgresql/blob/1a855aff87c7b86268dea87e79187dde1f45fa76/CMakeLists.txt#L32-L39

So there are no surprises to consumers

```suggestion
target_link_libraries(${PROJECT_NAME} PRIVATE oatpp::oatpp-postgresql)
```